### DISCUSSION TITLE
[Question]: Connection refused, ui collapse

### DISCUSSION THREAD

**Discussion by chengSenLong** [blakeblackshear/frigate#13348] [Ask A Question]
### What is your question?

After I configured docker-compose.yml, I ran docker compose up -d to check the logs and the UI page could not be opened.
<img width="1920" alt="440f40b14b9121a852050e08e986855" src="https://github.com/user-attachments/assets/ee7f6e6d-8561-4293-89a5-8f241d05a937">
<img width="1911" alt="2588f67dc2d7ca1ad3fcb9c4048059a" src="https://github.com/user-attachments/assets/c50a2b78-c5d3-4319-96ed-5314224788fa">
<img width="1920" alt="ca2db918da42b35a07c8d9273e72e7f" src="https://github.com/user-attachments/assets/1b6b09dc-fab4-454c-8a15-db03092210c6">

**Comment by hawkeye217** [comment]
Please fill out a new support request discussion with all of the required information so we can help.
